dev-cpp/pystring/pystring-1.1.3.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="C++ functions matching the interface and behavior of python string methods"
+HOMEPAGE="https://github.com/imageworks/pystring"
+
+if [[ "${PV}" == "9999" ]];  then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/imageworks/pystring.git"
+else
+	SRC_URI="https://github.com/imageworks/pystring/archive/v"${PV}".tar.gz -> "${P}".tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
```

Comment:
We'll have to drop keywords apart from ~amd64 for now and test it later on in a rekeywording bug.